cli/slsa-verifier/main.go

```diff
@@ -16,10 +16,6 @@ func check(err error) {
 	}
 }
 
-func ExperimentalEnabled() bool {
```

Comment:
Maybe we can keep this and disable the linter `//nolint: something`, since I suspect we'll use it again

Reply:
sgtm, agree